cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -240,6 +248,22 @@ public Network convert() {
         return network;
     }
 
+    private void addControlArea(Context context, CgmesControlAreaAdder cgmesControlAreaAdder, PropertyBag tf) {
+        String controlAreaId = tf.getId("ControlArea");
+        String controlAreaName = tf.getLocal("controlAreaName");
+        String energyIdentCodeEic = tf.getLocal("energyIdentCodeEic");
+        double netInterchange = tf.asDouble("netInterchange");
+        CgmesControlArea cgmesControlArea = cgmesControlAreaAdder.newCgmesControlArea(controlAreaId, controlAreaName, energyIdentCodeEic, netInterchange);
+
+        String terminalId = tf.getId("terminal");
+        Terminal terminal = context.terminalMapping().find(terminalId);
```

Comment:
I wonder if the terminal is linked to a switch (but I don't know if it is allowed indeed).

Reply:
It would only cause issue in one case for us: if the switch is inside a voltage level (so if a control area boundary is inside a voltage level) since a switch linking two voltage levels is a line in IIDM. So to investigate, but I don't think it will make problems.